views/pages/invite.njk

```diff
@@ -4,12 +4,21 @@
     <h1>{{ __('invite.title') }}</h1>
```

Comment:
The page title should be `Give this code to your doctor or nurse practitioner`.
We want to remove anything that could get in the way of our primary instruction to the applicant.

Reply:
done